README.md

```diff
@@ -185,6 +185,29 @@ data:
   password.txt: YXNkZmFkc2YgLW4K
 ```
 
+### Updating an existing Secret file
+
+#### Setup
+
+1. From the repo's `deploy` directory, run `gcloud auth application-default login`.
+1. A browser window will open.  Log in with your Google account.
+1. Your terminal will return saying that credentials have been saved.
+1. Run `unset GOOGLE_APPLICATION_CREDENTIALS`
+
+#### Decrypting
+
+1. open your `<app>.secret.sops.yml`, and grab the value for `sops/kms/gcp_kms/resource_id`
+1. `sops --decrypt --input-type yml --gcp-kms <resource_id> <env>/<app>.secret.sops.yml > <env>/<app>.secret.yml`
+  * Note that the encrypted file lacks the 'sops' portion of the name.
```

Comment:
"Note that the first filename in this command is the _encrypted_ file (i.e. with `.sops.`) and the second filename is the _decrypted_ file (i.e. without `.sops`)" is probably clearer since both filenames are in that command.